### ISSUE TITLE
Problem with special symbol in CoD2

### ISSUE THREAD

**Issue by BestNoob** [BigBrotherBot/big-brother-bot#397]
I cannot use B3 with CoD2 Call of duty 2 Server, all special symbols/signs got a flipped T in front of it.

I tried to decrypt this symbol and got different things:
 FQ 15 NAK

**Comment by BestNoob** [comment]
Okay, has to do with CoD2 itself but i dk why. **Is there anoother method to use commands for B3 ?**

**Comment by BobDylan10** [comment]
> I cannot use B3 with CoD2 Call of duty 2 Server, all special symbols/signs got a flipped T in front of it.
> 
> I tried to decrypt this symbol and got different things:
> � FQ 15 NAK

I'm not sure to get your problem, where does your server need special characters ? When you type a command ?

**Comment by 82ndab-Bravo17** [comment]
If you mean the symbol that appears on the line before what is actually typed then that is normal and B3 works fine with it there.

**Comment by BestNoob** [comment]
Yes everytime i want to use a command there is this smybol. But i had no problems with CoD4 only with CoD2.

**Comment by 82ndab-Bravo17** [comment]
Most likely you are not pointing to the correct log file then

**Comment by BestNoob** [comment]
Yep it was mp_server.log instead of never CoD titles games_mp.log

**Comment by 82ndab-Bravo17** [comment]
We don't support cracked servers.

**Comment by BestNoob** [comment]
But that was not the problem i think i got it.
EACH B3 Instance needs it OWN ! SQL Database.
Because there are no game specific flags/tags. So each B3 instance use the same info as the otehrs if you didnt created a seperate one.
